.github/workflows/tests.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
       - name: Cache models
         uses: actions/cache@v4
         with:
-          path: ~/.cache/huggingface
+          path: ~/.cache/huggingface/hub/
```

Comment:
This reduced it 25GB, but still too much :exploding_head: 

`Warning: Failed to save: Cache size of ~25675 MB (26922483338 B) is over the 10GB limit, not saving cache.`